### ISSUE TITLE
Plugin zfs_arc should not parse the header line name title data

### ISSUE THREAD

**Issue by m8t** [collectd/collectd#2954]
*   Version of collectd: 5.7.1
*   Operating system / distribution: Debian Stretch

## Expected behavior

Entry log about "data" should be ignored.

## Actual behavior

Plugin zfs_arc is logging at each poll the following line:
`parse_value: Failed to parse string as derive: "data".`

I suspect the plugin to `parse_value` the second line of that `arcstats` file `/proc/spl/kstat/zfs/arcstats` that contains the header "name type data":
```
# head /proc/spl/kstat/zfs/arcstats 
6 1 0x01 91 4368 6091888536 232566203459472
name                            type data
hits                            4    114750866
misses                          4    8620004
```

https://github.com/collectd/collectd/blob/master/src/zfs_arc.c#L243

Maybe I'm wrong, I don't have an env right now to build and test.

## Steps to reproduce

*   start collectd with plugin zfs_arc (need a valid arcstats file.)
*   watch logs

**Comment by rubenk** [comment]
First two lines should have been ignored:
https://github.com/collectd/collectd/blob/master/src/zfs_arc.c#L215

**Comment by rubenk** [comment]
But that is not the case in 5.7.1: https://github.com/collectd/collectd/blob/collectd-5.7.1/src/zfs_arc.c#L200. Any chance you could upgrade collectd?

**Comment by rubenk** [comment]
You'll need https://github.com/collectd/collectd/commit/5b0c0bef2694766cb7c18d2d57017bf89fc3847d#diff-97c2f2a5f84d1833513b7e810518bf0b which should be in 5.8

**Comment by rpv-tomsk** [comment]
Closed in #2975 and #2097.

Feel free to reopen if needed.

Thanks for using Collectd.

**Comment by m8t** [comment]
o/ Just for the record, it works fine with 5.8.0. Thanks for your quick reply and sorry for my delay.
